app/controllers/users_controller.rb

```diff
@@ -2,6 +2,7 @@ class UsersController < ApplicationController
   # Placeholder action for all users page
   def index
     @users = User.distinct
+    @placeholder_text = 'Placeholder text for all users'

```

Comment:
**OPTIONAL** - Good job done so far, it's not good practice to populate your controller with items that are not needed, please try to keep content in your controller as minimal as possible. I suggest you render this in the respective view file 👍 👍